modules/vba_documents/app/workers/vba_documents/upload_status_updater.rb

```diff
@@ -12,11 +12,9 @@ class UploadStatusUpdater
       unique_until: :success
     )
 
-    def perform
-      if Settings.vba_documents.updater_enabled
-        VBADocuments::UploadSubmission.in_flight.order(created_at: :asc).find_in_batches(batch_size: 100) do |batch|
-          VBADocuments::UploadSubmission.refresh_statuses!(batch)
-        end
+    def perform(submission_guids)
+      VBADocuments::UploadSubmission.where(guid: submission_guids).find_in_batches(batch_size: 100) do |slice|
```

Comment:
Do you think we should make batch size a constant?

Reply:
I like that idea! Will do

Reply:
resolved here: https://github.com/department-of-veterans-affairs/vets-api/pull/3177/commits/06b2ce13d9a7342ba8d6b77ab4c9ce52bd413f57